движение цм ла

nniksaaa opened the pull request:
по ду смоделировали движение цм ла
задали начальную скорость, константные значения и значения нагрузок
итог: получили график горизонтального полета ла в координатах ух

nniksaaa committed:
график полета

получили график горизонтального полета ла в координатах ух

nniksaaa committed:
инит

добавили скрипт с const значениями

nniksaaa committed:
точка с запятой

nniksaaa committed:
go to

сделали красиво

nniksaaa committed:
ошибка с х

исправили ошибку  х, теперь он увеличивается линейно

nniksaaa committed:
спид и nx

nniksaaa committed:
постоянная скорость ла

через пид регулятор установили постоянную скорость с переменным nx. определили это в отдельную сабсистем "controller"

nniksaaa committed:
Update train_model_la_nika.slx

NoMonsters commented:
А как запустить?

NoMonsters commented on Models/training/train_model_nika/sis_init.m:
```diff
@@ -0,0 +1,8 @@
+g = 9.8;
+ny = 1.5;
+%nx = 0;
+nZ = 0;
```
Назвать константы одинаково. Все с маленькой буквы черзе нижнее подчеркивание.

NoMonsters reviewed (changes requested):
По модели
* Сделать блоки одинакового норм размера (константы на основном уровне модели)
*  Вывести из модели Plane все сигнал о движении: угл эйлера, скорость, координаты. Подписать и залогировать их
* Сделать заданное значение скорости входным для регулятора
* Внутри модели Plane сделат ьвсех блоки входов одинаковыми. Вход nz странно выглядит.

Общая задача. Для проверки мне скинуть вариант: где самолет сбалансирован в горизонтальном полете на высота 1000 метров и скорости 100 м/с. Через 10 секунд после начала моделирования заданная скорость устанвливается на 130 м/с. Самолет выходит на эту скорость.

nniksaaa committed:
горизонтальный полет

горизонтальный полет в течении 10 сек со скоростью 100 м/с на высоте 1000м

nniksaaa committed:
новый параметр

добавили значение скорости, которое должно установиться после 10 секунд полета

nniksaaa committed:
готовая моделька

в первые 10 сек самолет в горизонтальном полете набирал скорость от v0 до speed_tgt, а другие 10 сек его скорость изменилась от speeed_tgt до speed_tgt_2

nniksaaa committed:
добавление

добавили выводы результатов о движении  из блока plane

NoMonsters reviewed (approved):
Нужно внести следующие изменения.
1. Удалить файлы если они не нужны:
train_model_nika.slx
train_model_nika2.slx
2. В целом окей. Давай тогда примем этот реквест в таком виде
3. Теперь сделай, пожалуйста эту модель с учетом угла крена и аналогичный регулятор по управлению скоростью движения в боковом канале
4. Эту модель уже размещай не в training а в plane с именем plan_имя
